src/inmanta/data/__init__.py

```diff
@@ -213,6 +228,48 @@ def ensure_boundary_type(
         else:
             return order_by_column_value
 
+    @property
+    def id_column(self):
+        return "id"
+
+    def as_start_filter(
```

Comment:
This needs a comment and this is way to much copy pasting. I had place it in seperate files and use diff to see the difference except for the slightly different variable names.